WORKSPACE

```diff
@@ -37,6 +37,15 @@ load("@io_bazel_rules_webtesting//web:repositories.bzl", "web_test_repositories"
 
 web_test_repositories(omit_bazel_skylib = True)
 
+# rules_python has to be placed before load("@io_bazel_rules_closure//closure:repositories.bzl")
+# in the dependencies list, otherwise we get "cannot load '@rules_python//python:py_xxx.bzl': no such file"
+http_archive(
+    name = "rules_python",
+    sha256 = "0a8003b044294d7840ac7d9d73eef05d6ceb682d7516781a4ec62eeb34702578",
+    strip_prefix = "rules_python-0.24.0",
+    url = "https://github.com/bazelbuild/rules_python/releases/download/0.24.0/rules_python-0.24.0.tar.gz",
```

Comment:
Let's also add a mirror URL? Instructions linked from b/133880558, feel free to ping for details.

Reply:
Done
Thanks for the link!